doc/languages.rst

```diff
@@ -169,9 +169,15 @@ to view the docstring of the syntax checker.  Likewise, you may use
          A list of include directories.  Relative paths are relative to the file
          being checked.
 
-      .. option:: flycheck-cppcheck-language-standard
+      .. option:: flycheck-cppcheck-standards
 
-         The C or C++ language standard to use via ``--std=``.
+         The C,  C++ and/or  POSIX standards  to use  via one  or more
+         ``--std=``.
+
+      .. option:: flycheck-cppcheck-suppressions
+
+         The  cppcheck  suppressions  list  to use  via  one  or  more
+         ``--suppress=``.
```

Comment:
@fredmorcos Oh, I didn't even know that this setting existed :flushed:

There's not a day on which I don't learn a new thing about Emacs :blush: